client/src/components/AssessmentTest.js

```diff
@@ -23,26 +24,253 @@ const [EmotionalRegulation, setEmotionalRegulation] = useState(0);
+    } else {
+      window.alert('Invalid number of answers');
+    }
+  };
+
+return (
+    <div>
+        <Form />
+        <div className="pagemainsettingassessment">
+<form onSubmit={handleSubmit} >
     <div>
         <h1>Assessment Test</h1>
-        <h3>Communication Skills</h3>
-        <p> Does your child use gestures, eye contact, and facial expressions appropriately for their age?</p>
-        <input type="radio" id="yes" name="communicationSkills" value="yes" onChange={e => setCommunicationSkills(e.target.value)} />
-    <label htmlFor="yes">Yes</label>
-    <input type="radio" id="no" name="communicationSkills" value="no" onChange={e => setCommunicationSkills(e.target.value)} />
-    <label htmlFor="no">No</label>
+       
+        <h3>Communication Skills</h3> 
+        <div>
+  <p>1. Does your child use gestures, eye contact, and facial expressions appropriately for their age?</p>
+  <input type="radio" id="yes-communicationSkills-1" name="communicationSkills" value="yes" onChange={e => setCommunicationSkills(e.target.value)} />
+  <label htmlFor="yes-communicationSkills-1">Yes</label>
+  <input type="radio" id="no-communicationSkills-1" name="communicationSkills" value="no" onChange={e => setCommunicationSkills(e.target.value)} />
+  <label htmlFor="no-communicationSkills-1">No</label>
+</div>
     </div>
     <div>
         <h3>Social Interaction</h3>
-        <p>Does your child engage in pretend play or imaginative activities?</p>
+        <div>
+        <p>2. Does your child engage in pretend play or imaginative activities?</p>
         <input type="radio" id="yes" name="socialInteraction" value="yes" onChange={e => setSocialInteraction(e.target.value)} />
     <label htmlFor="yes">Yes</label>
     <input type="radio" id="no" name="socialInteraction" value="no" onChange={e => setSocialInteraction(e.target.value)} />
+    <label htmlFor="no">No</label></div>
+    </div>
+    <div>
+        <h3>Repetitive Behavior</h3>
+        <div>
+        <p>3. Does your child engage in repetitive behaviors, such as hand-flapping or rocking?</p>
+        <input type="radio" id="yes" name="repetitiveBehavior" value="yes" onChange={e => setRepetitiveBehavior(e.target.value)} />
+    <label htmlFor="yes">Yes</label>
+    <input type="radio" id="no" name="repetitiveBehavior" value="no" onChange={e => setRepetitiveBehavior(e.target.value)} />
+    <label htmlFor="no">No</label></div>
+  </div>
+    <div>
+        <h3>Sensory Sensitivity</h3>
+        <div>
+        <p>4. Does your child have sensitivities to light, sound, touch, or textures? </p>
+        <input type="radio" id="yes" name="sensorySensitivity" value="yes" onChange={e => setSensorySensitivity(e.target.value)} />
+    <label htmlFor="yes">Yes</label>
+    <input type="radio" id="no" name="sensorySensitivity" value="no" onChange={e => setSensorySensitivity(e.target.value)} />
+    <label htmlFor="no">No</label>
+        </div> </div>
+<div>
+<h3> Language Development </h3>
+<div>
```

Comment:
Everything looks great!  I can tell you worked really hard on this!  I'ld love to know how you got this assessment working!
Would you consider implementing indentation here and in other places in your code?  For example:
<div>
      <h3> Language Development </h3>
</div>